descriptions.go

```diff
@@ -119,6 +121,33 @@ func (ad AttributeType) IsOptional() bool {
 	return ad.Optional == "true"
 }
 
+// Returns indices of random blind attributes within this credentialtype
+// The indices coincide with indices of an AttributeList (metadataAttribute at index 0)
+func (ct *CredentialType) RandomBlinds() []int {
+	indices := []int{}
+	for i, at := range ct.AttributeTypes {
+		if at.RandomBlind {
+			indices = append(indices, i+1)
+		}
+	}
+	return indices
+}
+
+func (ct *CredentialType) getAttributeTypeIdentifiers(indices []int) (ids []string) {
```

Comment:
Using `get` prefixes for getters is [not idiomatic in Go](https://golang.org/doc/effective_go.html#Getters) (also in `GetRandomBlindAttributeTypeIdentifers`).